scaleway/resource_iam_policy_test.go

```diff
@@ -160,6 +160,62 @@ func TestAccScalewayIamPolicy_NoUpdate(t *testing.T) {
 					resource.TestCheckResourceAttr("scaleway_iam_policy.main", "rule.0.organization_id", orgID),
 				),
 			},
+			{
+				Config: fmt.Sprintf(`
+					resource "scaleway_iam_application" "main" {
+						name = "tf_tests_app_basic"
+						description = "a description"
+					}
+
+					resource "scaleway_iam_policy" "main" {
+						name = "tf_tests_policy_noupdate"
+						description = "a description"
+						application_id = scaleway_iam_application.main.id
```

Comment:
Name of the test is `NoUpdate`, I don't expect it to create a resource and update it. Maybe move to another test